test/scenarios-periodics/el92-prel@el94-crel@optional-upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+	prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-prel-optionals
```

Comment:
We need new images:
- rhel-9.2-microshift-prel-optionals
- rhel-9.3-microshift-prel-optionals
- rhel-9.3-microshift-4.14
- rhel-9.4-microshift-crel-optionals
- rhel-9.4-microshift-source-optionals